data/operators.csv

```diff
@@ -47,3 +47,4 @@ Atria Convergence,ISP,signed + filtering,safe,24309,4053
 Gigabit DK,cloud,signed + filtering,safe,60876,7680
 The Wikimedia Foundation,cloud,signed + filtering,safe,14907,8716
 A2B Internet,ISP,signed + filtering,safe,51088,1755
+Telstra Australia,transit,signed,unsafe,1221,66
```

Comment:
As per [contribution docs](https://github.com/cloudflare/isbgpsafeyet.com/blob/master/CONTRIBUTING.md) AS1221 [AS rank is 76](https://asrank.caida.org/asns?asn=1221&type=search)

Consider updating to:
`Telstra Australia,transit,signed,unsafe,1221,76`

Reply:
Also update AS1221 to started.

`Telstra Australia,transit,started,unsafe,1221,76`